entity-service-impl/src/main/java/org/hypertrace/entity/query/service/converter/accessor/IOneOfAccessor.java

```diff
@@ -0,0 +1,11 @@
+package org.hypertrace.entity.query.service.converter.accessor;
+
+import java.util.Set;
+import org.hypertrace.entity.query.service.converter.ConversionException;
+
+public interface IOneOfAccessor<T, U extends Enum<U>> {
```

Comment:
As a practice, we don't prefix interfaces with `I` 
```suggestion
public interface OneOfAccessor<T, U extends Enum<U>> {
```